File: Find the Parity Outliner.py
def find_outlier(integers):
    even = 0
    odd = 0
    for i in integers:
        if i%2 == 0:
            even += 1
        else:
            odd += 1
    
    if(even > odd):
        for i in integers:
            if(i%2 != 0):
                return i
    else:
        for i in integers:
            if(i%2 == 0):
                return i

File: Find the Parity Outliner.js
function findOutlier(integers){
  var odd = [];
  var even = [];
  
  for(let i=0; i < integers.length; i++){
    if(integers[i]%2 == 0){
      even++;
    } else {
      odd++;
    }
  };
  
  if(even > odd){
    for(let j=0; j < integers.length; j++){
      if(integers[j]%2 != 0){
        return integers[j];
      }
    }
  } else {
    for(let j=0; j < integers.length; j++){
      if(integers[j]%2 == 0){
        return integers[j];
      }
    }
  }
  
}
